allennlp/modules/token_embedders/bag_of_word_counts_token_embedder.py

```diff
@@ -14,22 +15,36 @@ class BagOfWordCountsTokenEmbedder(TokenEmbedder):
     Each sequence gets a vector of length vocabulary size, where the i'th entry in the vector
     corresponds to number of times the i'th token in the vocabulary appears in the sequence.
 
+    By default, we ignore padding tokens.
+
     Parameters
     ----------
-    vocab_size: int
-        size of vocabulary
-    projection_dim : int, optional (default = None)
+    vocab: ``Vocabulary``
+    vocab_namespace: ``str``
+        namespace of vocabulary to embed
+    projection_dim : ``int``, optional (default = ``None``)
         if specified, will project the resulting bag of words representation
         to specified dimension.
+    ignore_oov : ``bool``, optional (default = ``False``)
+        if true, will set entry corresponding to OOV token to zero.
```

Comment:
I don't feel like this is a great description, especially if you're using a projection layer. Wouldn't it be more accurate to just say, "If true, we ignore the OOV token"?